src/ipc/ipc.c

```diff
@@ -401,15 +401,15 @@ int ipc_comp_connect(struct ipc *ipc,
 	/* check whether the components already exist */
 	icd_source = ipc_get_comp_by_id(ipc, connect->source_id);
 	if (!icd_source) {
-		tr_err(&ipc_tr, "ipc_comp_connect(): components already exist, connect->source_id = %u",
-		       connect->source_id);
+		tr_err(&ipc_tr, "ipc_comp_connect(): source components already not exist, source_id = %u sink_id = %u",
```

Comment:
"source component does not exist" and yes I am glad the strings are in English and not in Polish, Romanian or Mandarin !

Reply:
I've always heard learning new languages is valuable and allows to reach many things 😄